Metaverse/Config/DefaultGame.ini

```diff
@@ -0,0 +1,3 @@
+[/Script/EngineSettings.GeneralProjectSettings]
+ProjectID=D914E5863F4885D22EC07081DBEBD81C
```

Comment:
maybe ProjectID=PROJECT_ID and pass the token as env variable?

Reply:
It is a UID generated by https://docs.unrealengine.com/5.0/en-US/API/Runtime/Core/Misc/FGuid/. It should be fixed after project created. Let's leave it?

Reply:
SecurityToken could be passed by 
```
UnrealAndroidFileTool.exe -s AB187923123CD123 -p com.OrganizationName.ProjectName shell -k CCEED369EB42CBA717F8528399C6F4E9
```